po/es.po

```diff
@@ -109,269 +109,269 @@ msgid "Minimize or show previews"
 msgstr "Minimizar o mostrar previsualizaciones"
 
 #: Settings.ui.h:10
+msgid "Focus or show previews"
+msgstr "Focalizar o mostrar previsualizaciones"
```

Comment:
Maybe: `Enfoque o mostrar vistas previas`

Reply:
For the sake of parallelism, I'd rather use a verb form for both :) Also, in GNOME's terminology we use "previsualización" for "preview"